packages/contracts/contracts/factory/TokenFactory.sol

```diff
@@ -49,6 +49,10 @@ contract TokenFactory {
     /// @param token GovernanceWrappedERC20 token address
     event WrappedToken(GovernanceWrappedERC20 token);
 
+    /// @notice Thrown if token address is not ERC20.
+    /// @param token The token address
+    error TokenNotERC20(address token, bytes data);
```

Comment:
Aren't we we're supporting ERC20, 721 and 1155? Or is this only in regards of the DAO deposit/transfer?

Reply:
We don't support it yet.. Maybe in next versions.